modules/drivers/bigquery-cloud-sdk/test/metabase/driver/bigquery_cloud_sdk_test.clj

```diff
@@ -253,6 +257,27 @@
                    {:source-table (mt/id view-name)
                     :order-by     [[:asc (mt/id view-name :id)]]}))))))))
 
+(deftest sync-materialized-view-test
```

Comment:
Where did this come from?

Reply:
I'm not following the question. but I notice we don't have any sync materialized view test, so I added one.

Reply:
ah cool makes sense